extension-manifest-v3/src/background/debug.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+globalThis.ghostery = {};
```

Comment:
```suggestion
const debug = {};
globalThis.ghostery = debug;

export default debug;
```

Move file to `/utils/debug.js`